src/pages/home/report/LinkPreviewer.js

```diff
@@ -103,13 +104,14 @@ function LinkPreviewer(props) {
                             )}
                         </View>
                         {!_.isEmpty(title) && (
-                            <Text
+                            <TextLink
                                 fontSize={variables.fontSizeNormal}
                                 style={styles.pv2}
                                 color={colors.blueLinkPreview}
```

Comment:
I found that style variable `blueLinkPreview` was added with the link preview feature. So I guess the style of title is followed design doc and tend not to change the style here.

Reply:
Updated